components/match2/wikis/counterstrike/match_legacy.lua

```diff
@@ -44,6 +44,12 @@ function MatchLegacy.convertParameters(match2)
 		end
 	end
 
+	if match.winner == 0 then
+		match.winner = 'draw'
+	end
+
+	match.resulttype = nil
```

Comment:
shouldn't this be inside the if?

Reply:
No, that way I can avoid having resulttype set as  `draw`, `np` or `default`.